package.json

```diff
@@ -47,39 +47,43 @@
   },
   "homepage": "https://github.com/libp2p/js-interfaces#readme",
   "dependencies": {
-    "@types/bl": "^2.1.0",
+    "@types/bl": "4.1.0",
```

Comment:
has this lost the `^` for a reason?

Reply:
good catch, mistyped! I will PR this shortly